cmd/openshift-install/create.go

```diff
@@ -931,3 +937,34 @@ func IsSingleNode() (bool, error) {
 	}
 	return false, nil
 }
+
+// handleInterrupt returns a new context which will be cancelled with a user interrupt.
+// shutdown is an optional shutdown function that will be executed before the context
+// is cancelled.
+func handleInterrupt(signalCtx context.Context, shutdown func()) (context.Context, context.CancelFunc) {
+	ctx, cancel := context.WithCancel(context.Background())
+	logrus.RegisterExitHandler(cancel)
```

Comment:
Does this let the shutdown clean properly? From my understanding logrus exit handler (https://github.com/sirupsen/logrus/blob/v1.9.3/alt_exit.go#L54-L64) calls are made before the call to Exit(). We might want to pass a function that waits instead of the shutdown here?

Reply:
Makes sense! Will look at adding.